articles/quickstart/webapp/python/01-login.md

```diff
@@ -104,14 +108,14 @@ def callback_handling():
 
 ## Trigger Authentication
 
-Add a `/login` route that uses the `Authlib` client instance to redirect the user to the [login page](/hosted-pages/login).
+Add a `/login` route that uses the `Authlib` client instance to redirect the user to the [login page](/hosted-pages/login). Replace `YOUR_CALLBACK_URL` in the snippet below with the **Allowed Callback URL** configured for your Application in Auth0.
 
 ```python
 # /server.py
 
 @app.route('/login')
 def login():
-    return auth0.authorize_redirect(redirect_uri='YOUR_CALLBACK_URL', audience='https://${account.namespace}/userinfo')
```

Comment:
was this removed from the sample as well?

Reply:
I'm not seeing anything that matches this ... though I do see an audience mentioning userinfo here:

https://github.com/auth0-samples/auth0-django-web-app/blob/f2f5466311589acd954cd31e474baba5754d887e/01-Login/webappexample/settings.py#L145

Reply:
@joshcanhelp That link points to the "DJango" webapp. This is "Python" webapp.

The code is still referencing the userinfo audience. It [defines it here](https://github.com/auth0-samples/auth0-python-web-app/blob/master/01-Login/server.py#L30-L31) and [uses it here](https://github.com/auth0-samples/auth0-python-web-app/blob/master/01-Login/server.py#L93). I'm OK with removing this audience now if the ID token is going to be verified and the claims be returned to the user, as if it were a /userinfo request result. But please keep both sample and article in sync.

Reply:
Second reference should probably stay. Here's a PR for the first one:

https://github.com/auth0-samples/auth0-python-web-app/pull/52